include/boost/json/detail/impl/handler.ipp

```diff
@@ -51,8 +51,13 @@ bool
 handler::
 on_object_end(
     std::size_t n,
-    error_code&)
+    error_code& ec)
 {
+    if( !ignore_duplicate_keys )
+        ec = st.check_duplicates(n);
+    if( ec.failed() )
```

Comment:
you check `ec` even if `ignore_duplicate_keys` is true?

Reply:
Yeah, it's unnecessary